pkgmgr/config.go

```diff
@@ -66,6 +67,11 @@ func NewDefaultConfig() (Config, error) {
 			userConfigDir,
 			"cardano-up",
 		),
+		ContextDir: filepath.Join(
+			userDataDir,
+			"cardano-up",
+			"default",
```

Comment:
This is going to result in a path like `.../cardano-up/default/default/whatever` or `.../cardano-up/default/other_context_name/whatever`. We probably don't want that.

Reply:
It does not. I don't append to it, only overwrite the value.